bash-completion/trust

```diff
@@ -7,7 +7,7 @@ _trust()
 
     local commands command
 
-    commands='list extract extract-compat anchor dump'
+    commands='list extract extract-compat anchor dump check-format'
```

Comment:
Haven't checked, but does this auto-completes filename?

Reply:
I believe it should.